bodhi/validators.py

```diff
@@ -148,7 +148,7 @@ def validate_build_tags(request):
                 if not build_rel:
                     raise KeyError("Couldn't find release from build tags")
             except KeyError:
-                msg = 'Cannot find release associated with build: {}, tags: {}'.format(build, tags)
+                msg = 'Cannot find any tags associated with that nvr: {}, tags: {}'.format(build, tags)
```

Comment:
Fair enough. The same sentiment still remains though. I think having different error messages would be nicer than a single error message for both lines.